src/otx/v2/adapters/datumaro/pipelines/load_image_from_otx_dataset.py

```diff
@@ -42,7 +42,7 @@ def _get_unique_key(results: dict) -> tuple:
         if "cache_key" in results:
             return results["cache_key"]
         d_item = results["dataset_item"]
-        results["cache_key"] = d_item.media.path, d_item.roi.id
+        results["cache_key"] = d_item.media.path, d_item.annotations[0].id
```

Comment:
Do we have at least 1 annotation? It looks risky if not.
I'd rather recommend to use other unique attributes as the hash key elements.

Reply:
Actually, the second term of `cache_key` is not required because `cache_key` is exactly mapped to media and media path is always unique. I have just tried to preserve the current implementation but willing to fix this later (by removing second term for `cache_key`). Thank you.

Reply:
Alright, thanks for the clarification.